### DISCUSSION TITLE
The event will cause java.lang.NullPointerException

### DISCUSSION THREAD

**Discussion by larrying** [apache/skywalking#7530] [Q&A] [answered]
if the name of event is alarm, the parameter will be null, then it cause  java.lang.NullPointerException

![IMG_20210823_093957_edit_413805368453003](https://user-images.githubusercontent.com/31137045/130384246-af443fe8-cb89-4f42-a6d5-8e8dd4922163.jpg)
![IMG_20210823_095543_edit_413860364294140](https://user-images.githubusercontent.com/31137045/130384266-d4a07999-6854-41d4-b911-d6773d944215.jpg)

**Comment by wu-sheng** [comment]
I think you didn't submit the images.

**Reply by larrying** [reply]
> I think you didn't submit the images.

i will submit later

**Reply by larrying** [reply]
i have updated the images

**Comment by wu-sheng** [comment]
This seems an internal usage, could you share how you report event? And who/which service reports event?

**Comment by larrying** [comment]
it is not an internal usage, the alarm is automatically sent by the event

**Reply by kezhenxu94** [reply]
@larrying please post the exception stack trace

**Comment by larrying** [comment]
it is  the exception stack

![IMG_20210819_210836](https://user-images.githubusercontent.com/31137045/130388441-d615ddd8-cb59-40fa-bf07-96b7d12a804d.jpg)

**Comment by kezhenxu94** [comment, marked as answer]
Should be fixed at https://github.com/apache/skywalking/pull/7531
